package/boot/uboot-mvebu/Makefile

```diff
@@ -41,6 +41,7 @@ define U-Boot/espressobin
   BUILD_DEVICES:=globalscale_espressobin globalscale_espressobin-v7
   BUILD_SUBTARGET:=cortexa53
   UBOOT_CONFIG:=mvebu_espressobin-88f3720
+  DEFAULT:=n
```

Comment:
There's no point in disabling it here, this package will be selected by atf package and incomplete u-boot will still be present in supplementary files. The u-boot.mk has default target which does that. I think enabling/disabling this final installation target could be dependent on subtarget. This will look ugly but it'll work.

Reply:
The intention wasn't related to buildbot, just overwriting the default:
u-boot.mk currently defaults to enable u-boot builds for the selected board. So if you choose to build an OpenWRT image for espressobin, this is currently enabled. This overwrites it, so if someone builds for this device, u-boot for it isn't unnecessary enabled. Unless that someone chooses atf too of course.

With the other patch it will always be build on buildbot, so it wouldn't have an effect there.

Reply:
Ok, but useless u-boot.bin will still be present there which could be confusing.

Reply:
Right, I added a patch to disable installing u-boot.
I discovered that it's using the nice BuildTarget/bin common code for the installation, so I added another patch so the atf package uses it too.